.github/workflows/approved-for-ci-run.yml

```diff
@@ -88,15 +99,50 @@ jobs:
             Feel free to review/comment/discuss the original PR #${PR_NUMBER}.
           EOF
```

Comment:
We are leaving the file body.md but it is not committed to the branch.
Is it okay?

Reply:
`body.md` it's a PR's description; we use it only when we need to create a PR (via `gh pr create --body-file body.md ...`), and we're not supposed to commit it